IdentityCore/src/broker_operation/request/interactive_token_request/MSIDBrokerOperationInteractiveTokenRequest+InteractiveParameters.m

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) Microsoft Corporation.
+// All rights reserved.
+//
+// This code is licensed under the MIT License.
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files(the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and / or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions :
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+
+#import "MSIDBrokerOperationInteractiveTokenRequest+InteractiveParameters.h"
+#import "MSIDBrokerOperationTokenRequest+Parameteres.h"
+#import "MSIDBrokerOperationInteractiveTokenRequest.h"
+#import "MSIDInteractiveRequestParameters.h"
+
+@implementation MSIDBrokerOperationInteractiveTokenRequest (InteractiveParameters)
+
++ (instancetype)tokenRequestWithParameteres:(MSIDInteractiveRequestParameters *)parameters
+                                      error:(NSError **)error
+{
+    __auto_type request = [MSIDBrokerOperationInteractiveTokenRequest new];
+    BOOL result = [self fillRequest:request withParameteres:parameters error:error];
+    if (!result) return nil;
+    
+    request.accountIdentifier = parameters.accountIdentifier;
```

Comment:
would this also need claims and client capabilities, or those are just basic parameters and others will be added later?

Reply:
Just basic parameters. I will add other parameters later.